### PR TITLE
Findings base definition updates, attribute description updates

### PR DESCRIPTION
**Pull request by floydtree**
#### Description of changes:

1. Generalizing base event attribute descriptions to simplify usage in the findings base event.
2. Adjusting descriptions throughout classes.
3. Simplifying inheritance in findings classes.
4. Adding explicit requirement keys in the base event.

### CODE REVIEW

**Comment by pagbabian-splunk** [comment]
Thank you - looks good now.

**Review by pagbabian-splunk — APPROVED** [review]

File: events/findings/finding.json [hunk 1]
```diff
@@ -50,41 +50,37 @@
...
+          "caption": "New",
+          "description": "The Finding is new and yet to be reviewed."
+        },
+        "2": {
+          "caption": "In Progress",
+          "description": "The Finding is under review."
+        },
+        "3": {
+          "caption": "Suppressed",
+          "description": "The Finding was reviewed, determined to be benign or false positive and is now suppressed."
```
Comment (Aniak5) [inline events/findings/finding.json:77]: Nit: Add an `a` in front of `false positive`

File: events/findings/finding.json [hunk 2]
```diff
@@ -50,41 +50,37 @@
...
+        },
+        "4": {
+          "caption": "Resolved",
+          "description": "The Finding was reviewed and remediated and is now considered resolved."
```
Comment (Aniak5) [inline events/findings/finding.json:81]: Nit: This could be a bit more concise: `The finding was reviewed, remediated, and is now considered resolved.`

**Review by Aniak5 — APPROVED** [review]
LGTM!

**Review by mikeradka — APPROVED** [review]

**Review by zschmerber — APPROVED** [review]
Nice work, Looks good.

**Pull request merged**